tests/foreman/ui/test_user.py

```diff
@@ -913,25 +927,6 @@ def test_negative_delete_user(self):
             self.user.delete(user_name, really=False)
 
     @stubbed()
-    @tier2
-    def test_negative_delete_last_admin(self):
```

Comment:
Also with our current custom admin users approach it would lead to default admin user deletion, which is not cool.